lib/adapters/QUnitAdapter.js

```diff
@@ -58,27 +88,23 @@ export default class QUnitAdapter extends EventEmitter {
   }
 
   onTestDone (details) {
-    var test = this.idDataMap[details.testId]
-
-    if (details.failed !== 0) {
-      test.status = 'failed'
-    } else if (details.skipped === true) {
-      test.status = 'skipped'
-    } else {
-      test.status = 'passed'
-    }
+    var test
 
-    test.runtime = details.runtime
-    test.errors = this.errors
+    details.errors = this.errors
```

Comment:
I have done this to equal the MochaAdapter, here we can check in the convertTest function for `this.errors`